pepper-apis/src/main/java/org/broadinstitute/ddp/export/collectors/ActivityResponseCollector.java

```diff
@@ -334,6 +342,14 @@ private void collectComponentMappings(Map<String, Object> props, ComponentBlockD
             if (definition.getActivityType() == ActivityType.FORMS) {
                 flattenHeadersByOrderedDepthTraversal((FormActivityDef) definition);
             }
+            if (excludedFields != null) {
+                responseHeaders.removeAll(excludedFields);
```

Comment:
Should we remove from `deprecatedHeaders` too? Will there be a chance an exclude-field is in there? On the other hand, I'm not too sure how we'll handle first-fields for `deprecatedHeaders`... maybe it's safe to punt on first-fields?